geo-types/src/line.rs

```diff
@@ -165,6 +170,49 @@ impl<T: CoordinateType> From<[(T, T); 2]> for Line<T> {
     }
 }
 
+#[cfg(test)]
+impl<T> RelativeEq for Line<T>
+where
+    T: CoordinateType + Float + AbsDiffEq,
+{
+
+    #[inline]
+    fn default_max_relative() -> Self::Epsilon {
+        T::epsilon()
+    }
+
+    #[inline]
+    fn relative_eq(
+        &self,
+        other: &Line<T>,
+        epsilon: Self::Epsilon,
+        max_relative: Self::Epsilon,
+    ) -> bool {
+        self.start_point()
```

Comment:
I have fixed this in the commit, to this PR, labelled 'points.rs Simplify RelativeEq for Point'

Reply:
I want to respond to this question 

> With these traits implemented, we can now use something like:
> approx::assert_relative_eq!(line_string_1, line_string_2);
> Right?

I have added a new commit here "Added new feature relative_eq"
It does not work as I want yet... so there is more to come. 

But I am at the point where I should discuss some architectural re-plumbing in public. 

I actually want this to work in a module I am developing. This project rightly wants to keep approx as an internal dev-dependency. So If I want to see in in my third party module then I need to optionally expose approx when the other build environment ( Cargo.toml) enables  the geo module's feature "relative_eq" This might explain some of my changes.

My current issue is that I can ONLY demonstrate approx::assert_relative_eq working  in a limited number of build environments. It works in 'test' and 'doctest' ONLY and ONLY inside geo-types. When I try and demo this in geo sub-crate ( ie geo/src/algorithm/rotate.rs)  or my third party module.. then it fails to compile..